packages/utils/telemetry-utils/src/events.ts

```diff
@@ -23,16 +23,25 @@ export function safeRaiseEvent(
     }
 }
 
+/**
+ * Raises events pertaining to the connection
+ * @param logger - The logger to log telemetry
+ * @param emitter - The event emitter instance
+ * @param connected - A boolean tracking whether the connection was in a connected state or not
+ * @param clientId - The ID of the client
```

Comment:
nit: "The connected/disconnected clientId"  ("clientId" is a term in and of itself, IMO)

Reply:
Updated